### PR TITLE
ISC - Update network info - Requested updates

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* added Blast API info to networks-endpoints.mdx
* fixed additional info header in networks-endpoints.mdx
* added subgraphs.md article with link to blastApi
* added blastAPI.md article
* added iota_evm_dm.svg as icon @todo replace
* added announcement bar about IOTA EVM
* altered HomeLayout/index.tsx to add more focus to IOTA EVM
* made table headers vertical instead of horizontal in NetworkInfo/index.tsx
* removed unused import in networks-endpoints.mdx

Should be merged after #1541 

 ## Todo 

- [x] Update IOTA EVM thumbnail icon
- [ ] Update Oracles example to use IOTA EVM `contractAddress`, `feedIds`
- [ ] Add Supra code examples to Oracles (@trungtt198x ? )
- [ ] Update the style of the first EVM button on the homepage

## Links to any relevant issues

fixes https://github.com/iotaledger/devx/issues/402
fixes https://github.com/iotaledger/iota-wiki/issues/1554

## Type of change

- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Phyloiota** [comment]
I would suggest we update the sidebar section

I would recommend to either call the subheader "IOTA EVM" and "IOTA EVM Testnet"
Or to call the links/pages "IOTA EVM Explorer", "IOTA EVM Toolkit".

We should always make sure to clearly differentiate the EVM from the L1 and not call both "IOTA" IMHO, L1 is IOTA, L2 is IOTA EVM

**Comment by Phyloiota** [comment]
The page about Blast API is great; we just should keep in mind that we may not want to make a page introduction for every one of our future RPC provider. As we have a premium package with Blast this still makes sense, but as the EVM grows we will likely get further RPC providers on board (some chains have 8-12 of these) so we may want to rething merging those into a single page later

**Comment by Phyloiota** [comment]
The Subgraph page should be updated to include a reference to Goldsky https://goldsky.com/products/subgraphs

And in a few days we will also have a dedicated partner page here https://docs.goldsky.com/chains/supported-networks  which we then could link

**Comment by Phyloiota** [comment]
You can also add the dedicated Endpoints from Blast API for IOTA EVM and EVM Testnet to networks and endpoints (and maybe the Blast API article):

EVM Mainnet:
RPC: https://iota-mainnet-evm.public.blastapi.io
WSS: wss://iota-mainnet-evm.public.blastapi.io
Archive RPC: https://iota-mainnet-evm.blastapi.io/e7596858-fc63-4a54-8727-b885a2af4ec8

EVM Testnet:
RPC: https://iota-testnet-evm.public.blastapi.io
WSS: wss://iota-testnet-evm.public.blastapi.io
Archive RPC: https://iota-testnet-evm.blastapi.io/e7596858-fc63-4a54-8727-b885a2af4ec8

**Comment by Phyloiota** [comment]
What i am missing now in this version is the MultiSig Wallet section - where do we have that linked?

**Comment by lucas-tortora** [comment]
> What i am missing now in this version is the MultiSig Wallet section - where do we have that linked?

That's in another PR: https://github.com/iotaledger/iota-wiki/pull/1537

**Comment by Phyloiota** [comment]
Looks good now!

**Review by Phyloiota — APPROVED** [review]
nice job

File: docs/build/getting-started/subgraphs.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+description: 'Learn about the role of in subgraphs in smart, how they can provide indexed blockchain data to your smart contracts.'
```
Comment (Dr-Electron) [inline docs/build/getting-started/subgraphs.md:2]:
```suggestion
description: 'Learn about the role of subgraphs in smart contracts and how they can provide indexed blockchain data to your smart contracts.'
```

File: docs/build/getting-started/blastAPI.md
Comment (Dr-Electron) [inline docs/build/getting-started/blastAPI.md:1]: We should add frontmatter here

File: docs/build/getting-started/subgraphs.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+You can use the convenient Blast API on IOTA EVM to access [archival nodes](blastAPI.md#archival-nodes).
+
+:::
+
+:::tip Wasp Access Nodes
+
+You can find more information on how to connect to an access node in the
+[#nodesharing channel](https://discordapp.com/channels/397872799483428865/398600007378272256) on the [IOTA Discord](https://discord.iota.org/).
+
+:::
```
Comment (Dr-Electron) [inline docs/build/getting-started/subgraphs.md:99]: I don't know enough about subgraphs to evaluate this, but is this section needed here?

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx [hunk 3]
```diff
@@ -47,6 +48,66 @@ chain ID after deployment.**
...
+## IOTA EVM Tools
+
+The following tools are **only available on IOTA EVM**.
+
+### EVM Toolkit
+
+You can use the [IOTA EVM Toolkit](https://evm-toolkit.evm.iotaledger.net) to withdraw assets from IOTA EVM to IOTA L1.
+It also includes a wrapper IOTA <-> wIOTA.
+
+### MultiSig Walllets
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx:81]: We have multisig on both Shimmer and IOTA

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx [hunk 4]
```diff
@@ -47,6 +48,66 @@ chain ID after deployment.**
...
+If your project requires [Oracles](/build/oracles/) to provide data from the outside world, you find both Pyth and Supra have integrated IOTA EVM.
+
+<OraclesContractData />
+
+### Multicall3
+
+If you need to aggregate results from multiple contract reads into a single JSON-RPC request or execute multiple
+state-changing calls in a single transaction, you can use the [Multicall3 contract](https://explorer.evm.iota.org/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract).
+
+### Subgraphs
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx:97]: We have self hosted subgraphs on ShimmerEVM. Should we mention nakams fork here?

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx [hunk 5]
```diff
@@ -47,6 +48,66 @@ chain ID after deployment.**
...
+
+You can use the [IOTA EVM Toolkit](https://evm-toolkit.evm.iotaledger.net) to withdraw assets from IOTA EVM to IOTA L1.
+It also includes a wrapper IOTA <-> wIOTA.
+
+### MultiSig Walllets
+
+If you require and additional level of security, you can use the [Safe{} Wallet](https://safe.iotaledger.net/) as a
+Multisig solution on IOTA EVM.
+
+### Oracles
```
Comment (Dr-Electron) [inline docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx:86]: We have one oracle provider on ShimmerEVM too

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: docs/build/getting-started/subgraphs.md (hunk 2 above)
Comment (lucas-tortora) [reply docs/build/getting-started/subgraphs.md:99]: I think so, subgraphs need historical data to index and query past events and transactions. I think we have an article about them somewhere

File: docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx (hunk 4 above)
Comment (lucas-tortora) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx:97]: I think the article is a better place to do so, wdyt?

Comment (Dr-Electron) [reply docs/build/isc/v1.0.0-rc.6/docs/getting-started/tools.mdx:106]: Yeah sounds good to me 👍

File: docs/build/_partials/_oracles_contract_data.md
Comment (Dr-Electron) [inline docs/build/_partials/_oracles_contract_data.md:1]: Maybe we should add an admonition here or tabs as not both oracles are available on both networks. What do you prefer?

File: docs/build/_partials/_oracles_contract_data.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+|:----------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|     Pyth contract      | [https://explorer.evm.shimmer.network/address/0x290f23E4a034Db5237edCb5aA2D94Acb4DD19fD2](https://explorer.evm.shimmer.network/address/0x290f23E4a034Db5237edCb5aA2D94Acb4DD19fD2) |
+|  Supra Pull Contract   | [https://explorer.evm.shimmer.network/address/0xe41444462709484272F54371F3f53bBF900Ec49E](https://explorer.evm.shimmer.network/address/0xe41444462709484272F54371F3f53bBF900Ec49E) |
+| Supra Storage Contract | [https://explorer.evm.shimmer.network/address/0x3E5E89d14576cE9f20a8347aA682517fe65B4ACB](https://explorer.evm.shimmer.network/address/0x3E5E89d14576cE9f20a8347aA682517fe65B4ACB) |
+|  Supra Push Contract   | [https://explorer.evm.shimmer.network/address/0x3df842b27c997cEc63160E79CB4398c82645A1c3](https://explorer.evm.shimmer.network/address/0x3df842b27c997cEc63160E79CB4398c82645A1c3) |
```
[the remark this answers is not in the document]
Comment (Phyloiota) [reply docs/build/_partials/_oracles_contract_data.md:24]: So the Pyth contract was incorrect it seems. This is the one from their Docs:

https://explorer.evm.iota.org/address/0x8D254a21b3C86D32F7179855531CE99164721933

File: docs/build/_partials/_oracles_contract_data.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+import TabItem from '@theme/TabItem';
+
+<Tabs groupId='network'>
+<TabItem value='iota' label='IOTA EVM'>
+
+|     Contract Type      |                                                                           Contract Address                                                                           |
+|:----------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|     Pyth contract      | [https://explorer.evm.iota.org/address/0x8D254a21b3C86D32F7179855531CE99164721933](https://explorer.evm.iota.org/address/0x8D254a21b3C86D32F7179855531CE99164721933) |
+|  Supra Pull Contract   | [https://explorer.evm.iota.org/address/0x2FA6DbFe4291136Cf272E1A3294362b6651e8517](https://explorer.evm.iota.org/address/0x2FA6DbFe4291136Cf272E1A3294362b6651e8517) |
+| Supra Storage Contract | [https://explorer.evm.iota.org/address/0xD02cc7a670047b6b012556A88e275c685d25e0c9](https://explorer.evm.iota.org/address/0xD02cc7a670047b6b012556A88e275c685d25e0c9) |
```
Comment (Phyloiota) [inline docs/build/_partials/_oracles_contract_data.md:13]:
```suggestion
```

Comment (Phyloiota) [reply docs/build/_partials/_oracles_contract_data.md:13]: Pull and storage are the same contract, we can remove the mentioning of Storage and keep only Pull here

File: docs/build/_partials/_oracles_contract_data.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+</TabItem>
+<TabItem value='shimmer' label='ShimmerEVM'>
+
+|     Contract Type      |                                                                           Contract Address                                                                           |
+|:----------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|     Pyth contract      | [https://explorer.evm.shimmer.network/address/0x290f23E4a034Db5237edCb5aA2D94Acb4DD19fD2](https://explorer.evm.shimmer.network/address/0x290f23E4a034Db5237edCb5aA2D94Acb4DD19fD2) |
+|  Supra Pull Contract   | [https://explorer.evm.shimmer.network/address/0xe41444462709484272F54371F3f53bBF900Ec49E](https://explorer.evm.shimmer.network/address/0xe41444462709484272F54371F3f53bBF900Ec49E) |
+| Supra Storage Contract | [https://explorer.evm.shimmer.network/address/0x3E5E89d14576cE9f20a8347aA682517fe65B4ACB](https://explorer.evm.shimmer.network/address/0x3E5E89d14576cE9f20a8347aA682517fe65B4ACB) |
```
[the remark this answers is not in the document]
Comment (Phyloiota) [reply docs/build/_partials/_oracles_contract_data.md:23]: as above, we dont need the storage contract here

File: docs/build/_partials/_oracles_contract_data.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (Phyloiota) [reply docs/build/_partials/_oracles_contract_data.md:24]: Yes they are correct, can we also link to their Docs:

Pyth: https://docs.pyth.network/price-feeds/contract-addresses/evm
Supra Pull: https://gb-docs.supraoracles.com/docs/data-feeds/pull-model/networks
Supra Push: https://gb-docs.supraoracles.com/docs/data-feeds/decentralized/networks

Thanks

[the remark this answers is not in the document]
Comment (Phyloiota) [reply docs/build/_partials/_oracles_contract_data.md:24]: see my comments above

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**